docs/modules/ROOT/pages/how-tos/generic-env-vars.adoc

```diff
@@ -0,0 +1,25 @@
+= Generic Restic Environment Variables
```

Comment:
This file still doesn't show up in the menu. You have to actively link the page. Please add it to `docs/modules/ROOT/nav.adoc` (see e.g. https://github.com/k8up-io/k8up/blob/master/docs/modules/ROOT/nav.adoc)

you can view a preview of the documentation by running `make docs-preview` on `http://localhost:2020`.

Reply:
Ah true, I forgot it was an antora page :) I will add it

Reply:
Done